unaligned/align - cupid/cupid alignment

```diff
@@ -0,0 +1,29 @@
+**Cupid Alignment Examples**
```

Comment:
Not all Solos are listed, is this to avoid listing Limited roles?

Reply:
Yeah, I didn't want to list all limited roles, and for a normal game it's probably a bad idea to combine a limited solo with cupid anyway. I only listed Vampire because it has a special edge case, whereas the other limited teams don't.